db/migrate/20150330195256_create_reviews.rb

```diff
@@ -0,0 +1,10 @@
+class CreateReviews < ActiveRecord::Migration
+  def change
+    create_table :reviews do |t|
+      t.text :body
+      t.integer :user_id, null: false
+      t.integer :sitcom_id
+      t.integer :rating, null: false
```

Comment:
missing timestamps on reviews, going to need those. rake db:rollback and add them in